content/appeals/decision-reviews/release-notes/2025-08-08.md

```diff
@@ -0,0 +1,17 @@
+The Final Status (final_status) attribute has been added to the following evidence submission endpoints of the Decision Reviews API:
+
+-POST /services/appeals/v2/decision_reviews/notice_of_disagreements/evidence_submissions
+-GET /services/appeals/v2/decision_reviews/notice_of_disagreements/evidence_submissions/{uuid}
+-POST /services/appeals/v2/decision_reviews/supplemental_claims/evidence_submissions
+-GET /services/appeals/v2/decision_reviews/supplemental_claims/evidence_submissions/{uuid} 
+
+This attribute indicates whether the status of a submission is final. It will default to false unless one of the following conditions is met:
+
+- The submission reaches VBMS.
+- The submission has reached the success status but will not be uploaded into VBMS.
+- The submission expired.
+- The submission has errored, with a DOC1XX code.
+
+Submissions with a final_status of true will no longer update to a new status.
+
+To learn more, [review the Decision Reviews API documentation](https://developer.va.gov/explore/api/decision-reviews/docs?version=current).
```

Comment:
Should this be a relative link instead (`/explore/api/decision-reviews/docs?version=current`)? I think we try to use relative links on the portal.